packages/material-ui/src/TablePagination/TablePagination.js

```diff
@@ -67,7 +67,7 @@ export const styles = theme => ({
 });
 
 const defaultLabelDisplayedRows = ({ from, to, count }) => `${from}-${to} of ${count}`;
-const defaultRowsPerPageOptions = [10, 25, 50, 100];
+const defaultRowsPerPageOptions = [10, 25, 50, 100, -1];
```

Comment:
Why include -1 by default?

Reply:
Ah, forgot to remove this. My bad. Changing it right away.